web/src/app/schedules/ScheduleOverrideForm.js

```diff
@@ -97,6 +98,14 @@ export default function ScheduleOverrideForm(props) {
           <ScheduleTZFilter
             label={tz => `Configure in ${tz}`}
             scheduleID={scheduleID}
+            onChange={newZone => {
+              // update value in form container with new TZ
+              onChange({
+                ...value,
+                start: value.start.setZone(newZone),
```

Comment:
It was originally mapping the value on every change, but we cannot do this as it would change the input every time the user inputs into the field (i.e. CST -> EST toggle, if the user types in '8' in the hour field, '9' would appear instead -- this is unwanted behavior). We need to update the zone on the value only once, when the toggle is flipped